src/libNode/PoWProcessing.cpp

```diff
@@ -170,9 +170,10 @@ bool Node::StartPoW(const uint64_t& block_num, uint8_t ds_difficulty,
     auto checkerThread = [this]() mutable -> void {
       unique_lock<mutex> lk(m_mutexCVWaitDSBlock);
       if (cv_waitDSBlock.wait_for(
-              lk,
-              chrono::seconds(NEW_NODE_SYNC_INTERVAL + POW_WINDOW_IN_SECONDS +
-                              FALLBACK_EXTRA_TIME)) == cv_status::timeout) {
+              lk, chrono::seconds(NEW_NODE_SYNC_INTERVAL +
+                                  POW_WINDOW_IN_SECONDS + FALLBACK_EXTRA_TIME +
+                                  CONSENSUS_OBJECT_TIMEOUT)) ==
```

Comment:
It's better to use TX_DISTRIBUTE_TIME_IN_MS here rather than CONSENSUS_OBJECT_TIMEOUT. Which will be the real hard deadline for the node to receive DS block in time

Reply:
Ok, can change